integration_tests/src/main/python/cache_test.py

```diff
@@ -28,3 +31,48 @@ def test_passing_gpuExpr_as_Expr():
             .cache()
             .limit(50)
     )
+
+def test_cache_table():
+    spark.sql("CACHE TABLE range5 AS SELECT * FROM range(5)")
```

Comment:
Please don't use `spark` directly because you don't know the state it is in.  We don't reset the state between runs so it could be on the GPU or on the CPU there is no good way to know.  I am thinking of renaming it so it is a lot harder to get to.